src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -72,9 +58,28 @@ export class StackDetailsComponent implements OnInit {
     private logger: Logger
   ) { }
 
+  private setStackAnalysisRawData(): void {
```

Comment:
@joshuawilson That object seems to be accessed only from this component and hence making it private makes sense and therefore I think that should remain as it is. As making it as a model will result in that making public.

For moving private members below public ones, I have created an issue and I 'll handle that in that. - https://github.com/fabric8io/fabric8-ui/issues/146